options/ansi/generic/environment.cpp

```diff
@@ -106,7 +106,7 @@ char *getenv(const char *name) {
 	return const_cast<char *>(view.data() + s + 1);
 }
 
-int putenv(const char *string) {
+int __mlibc_putenv(char *string) {
```

Comment:
It would be nice to put this into the `mlibc::` namespace?

Reply:
Where would I put the declaration?

Reply:
Make some new ´environment.hpp` if nothing suitable exists?